index.php

```diff
@@ -1 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Cartoon Collection</title>
+</head>
+
+<body>
+
+<header>
+    <h1>My Cartoon Collection</h1>
+</header>
+
+
+
+
+
 <?php
+
+$db = new PDO('mysql:host=db;dbname=MyCollection', 'root', 'password'); // initialise the db connection
```

Comment:
Mike will want you to make some functions out of this code so you can test them; you should be able to get one out of the db construction, and another one for the foreach loop below. But as it stands it does work!